ingeniamotion/fsoe.py

```diff
@@ -92,6 +100,21 @@ def sto_activate(self) -> None:
         """Set the STO command to activate the STO"""
         self.__master_handler.dictionary.set(self.STO_COMMAND_UID, False)
 
+    def is_sto_active(self) -> bool:
+        """Check the STO state.
+
+        Returns:
+            True if the STO is active. False otherwise.
+
+        """
+        if self.__master_handler.state != StateData:
+            return True
+        fsoe_command = int.from_bytes(self.safety_slave_pdu_map.items[0].raw_data_bytes, "little")
```

Comment:
This should be a get() from a dictionary item, but since its Input and Output at the same time is not implemented.

Create issue or TODO for when this is done:
https://novantamotion.atlassian.net/browse/INGK-919